### ISSUE TITLE
Control Icons not showing

### ISSUE THREAD

**Issue by macasas** [evilz/vscode-reveal#1032]
I get the 4 control icons in VSCode preview
but after export to HTML I just have 4 outlined squares
Clicking each square works, but the colours are just squares, as are the up down arrows.

**Comment by macasas** [comment]
Yes, and the font awesome files. A full list is in the js console in the browser

**Comment by tsjshg** [comment]
Or you can see the icons on the exported HTML by adding the following 2 lines in your head of input markdown file.
<link href="https://use.fontawesome.com/releases/v6.6.0/css/all.css" rel="stylesheet">
<style> .reveal i.fa { font-family:FontAwesome; font-style: normal; } </style>

The first line loads the latest version of font-awesome, and the second one from [here](https://github.com/hakimel/reveal.js/issues/601#issuecomment-45993190).

But when I export the file into PDF, font-awesome icons disappear. So I can't use the icons for the contents body.

vscode-reveal==4.3.3
